src/foremast/templates/infrastructure/app_data.json.j2

```diff
@@ -24,10 +24,15 @@
                 {% if not loop.last %},{% endif %}
                 {% endfor %}
                 ]
-
-            }
+            },
+            "instanceLinks": [{
+                "title": "Internal Tools", "links": 
```

Comment:
I know @sijis called it Internal Tools, but I think `Instance Links` or just `Links` would be more clear to developers